services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedDeviceBackend.java

```diff
@@ -77,6 +80,16 @@ public MongoDbBasedDeviceBackend(
         return registrationService.readDevice(tenantId, deviceId, span);
     }
 
+    @Override
+    public Future<OperationResult<List<DeviceWithId>>> searchDevices(final String tenantId,
+            final int pageSize,
+            final int pageOffset,
+            final Optional<List<Filter>> filters,
+            final Optional<List<Sort>> sortOptions,
+            final Span span) {
```

Comment:
I have changed the searchDevices method signature to no longer use `Optional` around the filters and sort options. Can you please adapt your code accordingly?

Reply:
I have adapted the code according and pushed my changes.